pkg/build/strategies/layered/layered.go

```diff
@@ -145,6 +144,10 @@ func (builder *Layered) Build(config *api.Config) (*api.Result, error) {
 		return buildResult, fmt.Errorf("builder image uses ONBUILD instructions but ONBUILD is not allowed")
 	}
 
+	if config.BuilderImage == "" {
+		return nil, fmt.Errorf("repository name must have at least one component")
```

Comment:
yes - I'll change it.  What is there happens to be what was returned in this case before.